Add blog post for Tenzir Platform v1.7 release (#4893)
tenzir/tenzir: web/blog/tenzir-platform-v1.7/index.md

---
title: "Tenzir Platform v1.7: Explorer Drag'n'Drop"
slug: tenzir-platform-v1.7
authors: [lava]
date: 2025-01-06
tags: [release, platform]
comments: true
---

To kick off the new year, we're releasing [Tenzir Platform v1.7][github-release], featuring support for file drag and drop and a lot
of stability improvements..

![Tenzir Platform v1.7](tenzir-platform-v1.7.png)

[github-release]: https://github.com/tenzir/platform/releases/tag/v1.7.0

<!-- truncate -->

## Explorer Drag'n'Drop

> It is now possible to drag and drop JSON files to the explorer window in order to create a pipeline reading the data from the dropped file.

Reviewer: This is not restricted to JSON files. The feature supports any file through the new `from` operator in Tenzir Node v4.25.

---
title: "Tenzir Platform v1.7: Explorer Drag'n'Drop"
slug: tenzir-platform-v1.7
authors: [lava]
date: 2025-01-08
tags: [release, platform]
comments: true
---

To kick off the new year, we're releasing [Tenzir Platform v1.7][github-release], featuring support for file drag and drop and a lot

> of stability improvements..

Reviewer 2: ```suggestion
of stability improvements.
```

+ reflow

stability improvements.

![Tenzir Platform v1.7](tenzir-platform-v1.7.png)

[github-release]: https://github.com/tenzir/platform/releases/tag/v1.7.0

<!-- truncate -->

## Explorer Drag'n'Drop

It is now possible to drag and drop files to the explorer window in order to
create a pipeline reading the data from the dropped file.

The generated pipeline is using the [`from`
operator](/next/tql2/operators/from), so in order to use this feature a Tenzir

> Node with version 4.25 or later is required.

Reviewer: ```suggestion
Node with v4.25 or later is required.
```

All file types that are [automatically
recognized](/next/tql2/operators/from#file-extensions) by the `from` operator
can be used as source files.

## Webhook Analytics

> For sovereign edition users, it is now possible to specify a webhook URL as a

Reviewer: ```suggestion
For Sovereign Edition users, it is now possible to specify a webhook URL as a
```
